src/MessagingClient.ts

```diff
@@ -97,6 +111,19 @@ export class MessagingClient implements MessagingAPI {
         await this.matrixClient.sendReadReceipt(event)
     }
 
+    /** Mark all messages in the conversation as seen */
+    async markMessagesAsSeen(conversationId: ConversationId): Promise<void> {
+        const room = this.matrixClient.getRoom(conversationId)
+        if (!room) {
```

Comment:
Add a comment on why we are returning here clarifying room does not exist, we don't need to do anything.